src/Pickles/Pickles.UserInterface.UnitTests/CommandGeneration/WhenGeneratingPowerShellCommands.cs

```diff
@@ -0,0 +1,215 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="CLICommandGeneratorTests.cs" company="PicklesDoc">
+//  Copyright 2011 Jeffrey Cameron
+//  Copyright 2012-present PicklesDoc team and community contributors
+//  
+//  
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+
+using NFluent;
+
+using NUnit.Framework;
+
+using PicklesDoc.Pickles;
+using PicklesDoc.Pickles.UserInterface.CommandGeneration;
+using PicklesDoc.Pickles.UserInterface.Settings;
+
+namespace System.CommandGeneration
```

Comment:
The namespace doesn't fit with the rest of the project

Reply:
Fixed